### ISSUE TITLE
UI changes + configurability related to GeoBlacklight schema modifications

### ISSUE THREAD

**Issue by sgbalogh** [geoblacklight/geoblacklight#552]
Many of the issues being worked on during the 2017 sprint involve schema additions that will allow us to represent new types of data about our records (see #548). It's not clear what the UI effects of having this new metadata should be.

Things we need to figure out include:
- Representing multiple direct download links (is there still a concept of a "primary" download? can we configure organization in downloads drop down? should we make these things configurable?)
- Metadata tab look and feel
- Representing a codebook/variable-level metadata (also via a modal, similar to metadata tab?)
- and more!

We need to do some thinking about how to approach these issues, and (importantly) communicate any proposals to the broader GeoBlacklight community for feedback.

**Comment by ewlarson** [comment]
Item view - Just sharing a quick screen grab of the B1G development branch with IIIF support (with leaflet fullscreen option) and a bbox sidebar map.  It's not perfect, but it helps "orient" the end-user to what area the map depicts.  We have a lot of these in our collection.

[
![btaa_iiif_and_bbox](https://user-images.githubusercontent.com/69827/28529034-0be2c322-7055-11e7-8398-fbb7b288395a.png)
](url)
